src/main/java/bio/terra/app/controller/DatasetsApiController.java

```diff
@@ -248,6 +250,20 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
     return jobToResponse(jobService.retrieveJob(jobId, userReq));
   }
 
+  @Override
+  public ResponseEntity<BulkLoadHistoryModelList> getLoadHistoryForLoadTag(
+      @PathVariable("id") UUID id,
+      @PathVariable("loadTag") String loadTag,
+      @Valid @RequestParam(value = "offset", required = false, defaultValue = "0") Integer offset,
+      @Valid @RequestParam(value = "limit", required = false, defaultValue = "10") Integer limit) {
```

Comment:
Hmm. I was following the example of `enumerateDatasets`. I seem to remember something about the interface method declaration annotations for default values not propagating to the implementing method.

Reply:
It would be nice to know for sure. Could you try removing it and see if the endpoint works as expected? When I was working on the CDA I didn't copy these annotations to my endpoint implementation and it seems to work fine.